browser/css/jsdialogs.css

```diff
@@ -1251,7 +1251,7 @@ input[type='number']:hover::-webkit-outer-spin-button {
 
```

Comment:
Maybe we could add a default that is the same as the max for the list so the whole dialog doesn't flick/changes:

```css
#zoterolist {
	width: 800px;
}
```

Reply:
done